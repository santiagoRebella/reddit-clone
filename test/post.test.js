const React = require('react');
const Post = require('components/post');
const renderer = require('react-test-renderer');

test('Post is rendered correctly', () => {
  const component = renderer.create(
    <Post
        key="someid"
        title="Test title"
        description="Test Description"
        url="Test Url"
        image="Test image"
        author="Test author"
        time={1234567} />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();

});
