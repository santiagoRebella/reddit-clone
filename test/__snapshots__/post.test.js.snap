// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Post is rendered correctly 1`] = `
<div
  className="card"
  style={
    Object {
      "display": "inline-block",
      "margin": 10,
      "width": 240,
    }
  }
>
  <img
    alt="Post image cap"
    className="card-img-top"
    src="Test image"
  />
  <div
    className="card-body"
  >
    <h6
      className="card-title"
    >
      author: 
      Test author
    </h6>
    <p
      className="card-text"
    >
      created: 
      January 15th 1970, 6:56:07 am
    </p>
    <h5
      className="card-title"
    >
      Test title
    </h5>
    <p
      className="card-text"
    >
      Test Description
    </p>
    <a
      className="btn btn-primary"
      href="https://reddit.comTest Url"
    >
      Comments
    </a>
  </div>
</div>
`;
